packages/cli/completions/wm.bash

```
@@ -47,7 +51,7 @@ _wm_completion() {
       ;;
 
     modify)
-      opts="${opts} --id --type --raise --important --no-signal --content --write --interactive --json --jsonl"
+      opts="${opts} --id --type --raise --mark-starred --clear-signals --content --write --interactive --json --jsonl"
       COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)